tabulator/parsers/ndjson.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import jsonlines
+
+from .. import helpers
+from . import api
+
+
+# Module API
+
+class NDJSONParser(api.Parser):
+    """Parser to parse NDJSON data format.
+
+    See: http://specs.okfnlabs.org/ndjson/
+    """
+
+    # Public
+
+    options = [
+    ]
+
+    # XXX: I would suggest to move this whole boilerplate to api.Parser, most
```

Comment:
Good thing about interface implementation concept - all parsers are fully independent. So it's much more flexible in terms of maintaining, refactoring, writing new ones (no need to learn base class internal state API) etc. I suppose lately programming world has a tendency to avoid inheritance as much as possible changing it to less problematic concepts like `go`, `rust` etc do.